aptos-move/aptos-aggregator/src/aggregator_extension.rs

```diff
@@ -241,11 +257,9 @@ impl Aggregator {
         // In theory, any delta will be applied to existing value. However,
         // something may go wrong, so we guard by throwing an error in
         // extension.
-        let value_from_storage = resolver
-            .resolve_aggregator_value(id, AggregatorReadMode::Precise)
```

Comment:
Why is AggregatorReadMode removed? How are we going to have two different read modes now?

Reply:
The method with different modes has a default unimplemented!(...) because this is a main branch, so having unused _mode seemed a bit ugly :)  once we rebase on aggregators branch, we can have a non-default implementation and get rid of V1.

Btw, V1 would also use cheap and precise reads? Or V2 only?

Reply:
That's the AggregatorResolver design question IMO. In the extension, we combine the representation, in executor it's fully split, so it's up to the resolver to determine where to fall on the spectrum.
V1 would only use precise reads, I believe, there is no use for LastCommitted, and Delta can always be unset, as soon as (aggregated/expensive/precise) read happens, it becomes Data. I think having V1 resolution API separate can allow using a specific type (StateKey directly, as opposed to new enum AggregatorID and then having to assert the inner variant) and to avoid checking that V1 isn't trying to do a cheap read. However, not a strong opinion, and wonder what @igor-aptos thinks.

Reply:
I don't have a strong opinion, seems fine to have separate getters for V1 and V2 (V2 will have to return union of u128 and String anyways).
maybe we decide to refactor after everything is connected.